src/util/stream_utils.js

```diff
@@ -30,19 +30,20 @@ function get_tap_stream(func) {
 }
 
 const async_pipeline = util.promisify(stream.pipeline);
-async function pipeline(streams) {
+async function pipeline(streams, reuse_last_stream) {
     if (!streams || !streams.length) throw new Error('Pipeline called without streams');
     if (streams.find(strm => strm.destroyed)) {
         const err = new Error('Pipeline called on destroyed stream');
         for (const strm of streams) {
-            if (!strm.destroyed) strm.destroy(err);
+            if (!strm.destroyed) strm.destroy();
```

Comment:
@romayalon Isn't it just enough to try-catch around this call?

Reply:
try and catch won't be enough, but we can listen to the error event on the stream: 
```
const fooErr = new Error('foo error');
myStream.destroy(fooErr);
myStream.on('error', (fooErr) => console.error(fooErr.message)); // foo error****
```

but in the next line, we are throwing the same error so I'm not sure it's necessary to emit it and listen, WDYT?